File: python class exercises/Assignment1 python/age.py
#prompt two user to enter their age 
#collect the two user's age ranging from 1 to 80
#calculate how many years ago that the first user input is twice as old as the second user input



fathers_age = int(input("Enter your age papa junior(from 1 - 80) :"))
son_age = int(input("Enter your age junior(from 1 -80)"))

calculate_years_ago = son_age * 2

if(fathers_age < calculate_years_ago):
	print(calculate_years_ago - fathers_age)
else:
    print(fathers_age - calculate_years_ago )

File: python-class-exercises/Assignment1 python/Age.java
/*prompt two user to enter their age 
#collect the two user's age ranging from 1 to 80
#calculate how many years ago that the first user input is twice as old as the second user input and check the first user input on how old he would be if he is twice as old as his son
 



import java.util.Scanner;
public class Age{
	public static void main(String [] args){
       
        Scanner input = new Scanner(System.in);
        
     System.out.print("Enter your age papa chinedu: ");
      int fatherAge = input.nextInt();

	 System.out.print("Enter your age chinedu: ");
      int sonAge = input.nextInt();

	 int fatherAgeTwice = sonAge * 2
 
	if(fatherAge < fatherAgeTwice){
             System.out.printf("Father age will be %d if he is twice as old as his son",(fatherAge - fatherAgeTwice));
         }

	else{
           System.out.printf("%d years ago the father was twice as old as his son",(fatherAgeTwice - fatherAge));
	}

  }
}